### ISSUE TITLE
NVDA does not work

### ISSUE THREAD

**Issue by iyar-avital** [woltapp/wolt_modal_sheet#330]
## Bug report
When opening a dialog with Wolt Modal, NVDA gets the focus of the dialog but cannot read the widgets inside it

### Steps to reproduce

1. Open a dialog 
2. Start NVDA 
3. Try moving to the next item Inside the dialog
4. NVDA reads "Button dialog"

### Expected behavior
Accessibility will operate on the widgets and read the semantics

**Comment by ulusoyca** [comment]
@iyar-avital Yes, we are aware of this and working on a solution. Our next update is focusing on fixing this (using pages as widget)

**Comment by iyar-avital** [comment]
Any update? @ulusoyca
